packages/stack/APKBUILD

```diff
@@ -0,0 +1,24 @@
+# Contributor: Markus Schirp <mbj@schirp-dso.com>
+# Maintainer: Markus Schirp <mbj@schirp-dso.com>
+arch=x86_64
+license=BSD-3-Clause
+pkgdesc='The Haskell Tool Stack'
+pkgname=stack
+pkgrel=0
+pkgver=2.1.3
+dirname="stack-$pkgver-linux-x86_64-static"
+filename="$dirname.tar.gz"
+source="https://github.com/commercialhaskell/stack/releases/download/v$pkgver/$filename"
+url=https://www.haskellstack.org
+
+package() {
+  cd "$srcdir/$dirname"
+
+  install -Dm755 stack "$pkgdir/usr/bin/stack"
```

Comment:
Instance of https://github.com/mbj/apk-build/pull/1/commits/820343cc510bcea80ee040f1a890e2a93ef2dd8a#r387981303